CMakeLists.txt

```diff
@@ -51,17 +52,78 @@ endif()
 
 PROJECT(OpenSceneGraph)
 
-SET(OPENSCENEGRAPH_MAJOR_VERSION 3)
-SET(OPENSCENEGRAPH_MINOR_VERSION 4)
-SET(OPENSCENEGRAPH_PATCH_VERSION 0)
-SET(OPENSCENEGRAPH_SOVERSION 130)
+
+set(
+    ROGII_FOLDER_PATH
+    "${CMAKE_CURRENT_LIST_DIR}/rogii"
+)
+
+include(${ROGII_FOLDER_PATH}/Version.cmake)
+include(${ROGII_FOLDER_PATH}/npm/Include.cmake)
+include(${ROGII_FOLDER_PATH}/environment.cmake)
+
+include(${ROGII_FOLDER_PATH}/utils/SetupRpathLink.cmake)
+include(${ROGII_FOLDER_PATH}/utils/Rpath.cmake)
+
+if(OSG_QT_ANGLE AND MSVC)
+	CNPM_ADD_PACKAGE(
```

Comment:
If I remember correctly, we only need Qt if OSG_QT_ANGLE is set and only for Windows. But we need msvs and WindowsSDK for Windows and gxx_runtime for linux anyway. Also you have these definitions in environment_linux.cmake and environment_windows.cmake. Please put things in order.